### PR TITLE
chore(FDS-342): Add test for email input field

### PR DESCRIPTION
**Pull request by rociobaena**
This PR includes:
Fixture for DataEmail module
Unit test

### Snapshots
Snapshot for DataEmail test

## New Component Checklist

- [x] Includes unit tests for _ALL_ required FDS use cases
- [x] Does not mute any top level API props in React
- [x] Includes an explanation for any changes to Jest snapshots (should be a PR tag automatically added if this happens)
- [ -] Component uses `React.forwardRef()` when a single DOM node is returned, _AND_ it makes React-sense to access that DOM node

### CODE REVIEW

**Review by lordpixel — DISMISSED** [review]
👍

File: packages/cascara/src/modules/DataEmail/DataEmail.js [hunk 1]
```diff
@@ -28,13 +29,14 @@ const DataEmail = ({
   ...rest
 }) => {
   const { isEditing, formMethods } = useContext(ModuleContext);
+  const { setAriaLabel, setHtmlFor } = getHTMLLabels(isLabeled, label);
```
Comment (rociobaena) [inline packages/cascara/src/modules/DataEmail/DataEmail.js:32]: @brianespinosa I'm not sure about this name. Could you advise here?

Comment (brianespinosa) [reply packages/cascara/src/modules/DataEmail/DataEmail.js:32]: I think `getAccessibleLabelSetters`

**Review by brianespinosa — COMMENTED** [review]
Let me know when that function helper is changed and this is ready to go!

**Review by brianespinosa — APPROVED** [review]
🙌🏽

**Pull request merged**